config/example/config.txt

```diff
@@ -435,3 +435,8 @@ API_RATE_LIMIT 50
 ## IPs that should not be throttled by the API RATE Limiter
 ## IPs are separated by ;
 API_RATE_LIMIT_WHITELIST 127.0.0.1
+
+## GELF Logging Configuration
+LOG_GELF_ENABLED 1

```

Comment:
Example, perhaps chuck this to 0? I think that's what's making Travis fail, as it'll try to gelf and fails.